tests/site-nav.js

```diff
@@ -46,11 +46,11 @@ describe('SiteNav', function() {
 
   describe('constructor()', function() {
     it('should set body to jqueryized selector', function() {
-      expect(this.siteNav.$body).to.be.ok;
-      expect(this.siteNav.$body.length).to.be.ok;
```

Comment:
Oh, I mean that these lines could stay as is, having the same test for `siteNav.$body` _and_ for `siteNav.$element`.